### PR TITLE
Add gas usage graph to energy.markdown

### LINKED ISSUE #19188: Documentation for Gas Usage Card missing
### Feedback

With the addition of the gas usage graph in the energy dashboard, I would expect it also pops up here to be include in other dashboards. 

### URL

https://www.home-assistant.io/lovelace/energy/

### Version

2021.9.1

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jelmerkk**
Added energy-gas-graph with screenshot

## Proposed change
Add the gas usage graph to the documentation

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: #19188

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @jelmerkk, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#19190).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_lovelace/energy.markdown [hunk 1]
```diff
@@ -52,6 +52,20 @@ The solar production graph card shows the amount of energy your solar panels hav
 type: energy-solar-graph
 ```
 
+## Gas usage graph
+
+<p class='img'>
+  <img src='/images/lovelace/energy/gas-usage-graph.png' alt='Screenshot of the gas usage graph card'>
+  Screenshot of the Gas usage graph card.
```
Comment (ludeeus) [inline source/_lovelace/energy.markdown]:
```suggestion
  Screenshot of the gas consumption graph card.
```

File: source/_lovelace/energy.markdown [hunk 2]
```diff
@@ -52,6 +52,20 @@ The solar production graph card shows the amount of energy your solar panels hav
...
+</p>
+
+The gas usage graph card shows the amount of gas consumed per source.
```
Comment (ludeeus) [inline source/_lovelace/energy.markdown]:
```suggestion
The gas graph card shows the amount of gas consumed per source.
```

File: source/_lovelace/energy.markdown [hunk 3]
```diff
@@ -52,6 +52,20 @@ The solar production graph card shows the amount of energy your solar panels hav
 type: energy-solar-graph
 ```
 
+## Gas usage graph
+
+<p class='img'>
+  <img src='/images/lovelace/energy/gas-usage-graph.png' alt='Screenshot of the gas usage graph card'>
```
Comment (ludeeus) [inline source/_lovelace/energy.markdown]:
```suggestion
  <img src='/images/lovelace/energy/gas-usage-graph.png' alt='Screenshot of the gas consumption graph card'>
```

File: source/_lovelace/energy.markdown [hunk 4]
```diff
@@ -52,6 +52,20 @@ The solar production graph card shows the amount of energy your solar panels hav
 type: energy-solar-graph
 ```
 
+## Gas usage graph
```
Comment (ludeeus) [inline source/_lovelace/energy.markdown]:
```suggestion
## Gas consumption graph
```

Comment (ludeeus) [reply source/_lovelace/energy.markdown]:
```suggestion
## Gas graph
```

File: source/_lovelace/energy.markdown [hunk 5]
```diff
@@ -52,6 +52,20 @@ The solar production graph card shows the amount of energy your solar panels hav
...
+
+<p class='img'>
+  <img src='/images/lovelace/energy/gas-usage-graph.png' alt='Screenshot of the gas usage graph card'>
+  Screenshot of the Gas usage graph card.
+</p>
+
+The gas usage graph card shows the amount of gas consumed per source.
+
+#### Example
+```yaml
```
Comment (ludeeus) [inline source/_lovelace/energy.markdown]:
```suggestion
#### Example

```yaml
```

File: source/_lovelace/energy.markdown (hunk 3 above)
Comment (ludeeus) [reply source/_lovelace/energy.markdown]: The filename should be `gas-graph.png`

**Comment by jelmerkk** [comment]
Thanks for the suggestions @ludeeus, done

**Review by ludeeus — APPROVED** [review]
Thanks :+1:

**Pull request merged**